testing/load-aware/command_args.yaml

```
@@ -23,3 +23,16 @@ load_aware scale_test:
   duration: {{ load_aware.scale_test.duration }}
   instances: {{ load_aware.scale_test.instances }}
   namespace: {{ load_aware.scale_test.namespace }}
+  scheduler: {{ load_aware.scale_test.scheduler }}
+
+utils build_push_image/deps:
+  namespace: {{ load_aware.scale_test.namespace }}
+  image_local_name: coreutils-deps
+  tag: latest
```

suggestion:
```suggestion
  image_local_name: coreutils
  tag: deps
```
(`latest` isn't recommenced anymore as container image tag)

---

Ah ok, that makes sense. I'll make that swap!